metadatastore/commands.py

```diff
@@ -1165,3 +1165,46 @@ def find_last(num=1):
         if next(c) == num:
             raise StopIteration
         yield _cache_run_start(rs)
+
+
+def get_header(run_start, allow_no_run_stop):
```

Comment:
Why are we giving a different meaning to "header" in mds and "header" in dataportal?

I.e., 
```python
db[-1] != mds.get_header(list(mds.find_last()[0]))
```